subprojects.yaml

```diff
@@ -0,0 +1,56 @@
+subprojects:
+- name: Helm Core
+  ownersLink: https://raw.githubusercontent.com/helm/helm/master/OWNERS
+  repos:
+  - https://github.com/helm/helm
+- name: Charts
+  owners:
+  - lachie83
+  - prydonius
+  - sameersbn
+  - viglesiasce
+  - unguiculus
+  - scottrigby
+  - mattfarina
+  - davidkarlsen
+  - paulczar
+  - cpanato
+  - jlegrone
+  - maorfr
+  emeritus:
+  - foxish
+  - linki
+  - mgoodness
+  - seanknox
+  repos:
+  - https://github.com/helm/chart-releaser-action
+  - https://github.com/helm/charts-repo-actions-demo
+  - https://github.com/helm/chart-testing-action
+  - https://github.com/helm/chart-testing
+  - https://github.com/helm/chart-releaser
+  - https://github.com/helm/homebrew-tap
+  - https://github.com/helm/kind-action
+  - https://github.com/helm/hub
+- name: Website
+  ownersLink: https://raw.githubusercontent.com/helm/helm-www/master/OWNERS
+  repos:
+  - https://github.com/helm/helm-www
+- name: Community Management
+  owners:
+  - karenhchu
```

Comment:
Should the helm/community repo bee listed here or is it something different?

Reply:
The community repo is managed by the org maintainers right now. Karen is doing other things.

Reply:
@marckhouzam or anyone else... what would be a good way to document the community repo being owned by the org maintainers? Any suggestions?

Reply:
How about
`ownersLink: https://github.com/helm/community/blob/master/MAINTAINERS.md`